nearcore/src/runtime/mod.rs

```diff
@@ -218,8 +221,18 @@ impl NightshadeRuntime {
         genesis: &Genesis,
         tracked_config: TrackedConfig,
         runtime_config_store: RuntimeConfigStore,
+        num_epochs_to_keep_store_data: Option<u64>,
     ) -> Self {
-        Self::new(home_dir, store, genesis, tracked_config, None, None, Some(runtime_config_store))
+        Self::new(
+            home_dir,
+            store,
+            genesis,
+            tracked_config,
+            None,
+            None,
+            Some(runtime_config_store),
+            num_epochs_to_keep_store_data.unwrap_or(MIN_NUM_EPOCHS_TO_KEEP_STORE_DATA),
```

Comment:
If there was `max` is `new` method as suggested above, this could be `.unwrap_or_deafult()`. Though I’d consider just making the argument `u64` rather than `Option<u64>`. Not a huge matter considering this is only for testing I suppose.

Reply:
I was mainly trying to avoid importing the constant everywhere.